documentation/user/Installation.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: wiki
+title: Installation
+description: Step-by-step guide for installing ACE3
+group: user
+order: 0
+parent: wiki
+---
+
+## 1. Overview
+This guide will explain how to get ACE3 up and running faster than you can say banana.
+
+## 2. Prerequisites
+- Download [CBA](http://www.armaholic.com/page.php?id=18767).
+- Download [ACE3](https://github.com/acemod/ACE3/releases/tag/v3.0.1).
```

Comment:
https://github.com/acemod/ACE3/releases/latest should point to most recent release without needing to be updated each time.